examples/Jaeger.Example.WebApi.NetFx/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Jaeger.Example.WebApi.NetFx")]
```

Comment:
Fill with data from `Directory.Build.props` if not possible to directly include.

Reply:
I am not sure I got this right, never use `Directory.Build.props` before